spec/models/step_by_step_page_spec.rb

```diff
@@ -192,6 +192,30 @@
     end
   end
 
+  describe 'links_checked_since_last_update?' do
+    let(:step_by_step_with_step) { create(:step_by_step_page) }
```

Comment:
Could this create a `step_by_step_page_with_steps` so a step doesn't need to be created in every step?